docs/advanced_tutorials/projectors.md

```diff
@@ -0,0 +1,20 @@
+This section introduces the `ProjectorBlock` as an implementation for the the quantum mechanical projection operation. It evaluates to the outer product of a ket and a bra expressed as bitstrings.
```

Comment:
Can you add the mathematical formula and a little bit more explanation of the interface?

Reply:
Done.